lib/commands/message/verify.js

```diff
@@ -1,7 +1,80 @@
 'use strict'
 
-const chalk = require('chalk')
+const Joi = require('joi')
+const arkjs = require('arkjs')
+const crypto = require('crypto')
+const network = require('../../services/network')
+const output = require('../../utils/output')
+const networks = require('../../config/networks')
+const schema = {
+  msg: Joi.string().required(),
+  publickey: Joi.string().required(), // TODO hex?
```

Comment:
it should be `publicKey` and `hex` for validation

Reply:
thx, I'll update that